app/main.py

```diff
@@ -32,13 +36,13 @@ def main() -> None:
     ]
 
     # run the programs
-    service.run_program(wake_up_program)
-    service.run_program(sleep_program)
+    await service.run_program(wake_up_program, run_in_parallel=True)
+    await service.run_program(sleep_program, run_in_parallel=True)
```

Comment:
you cannot run all programs in parallel
what if play song starts earlier than speaker is switched on?
or clean is run earlier than flash?
The order matters